res/TemplateBatchFiles/libv3/models/protein/empirical.bf

```diff
@@ -496,6 +506,180 @@ function models.protein.mtMAMML.ModelDescription(type) {
 
 
 
+/**************************************** cpREV functions *************************************/
```

Comment:
(Thinking out loud) This code is starting to look super redundant. There has to be a way to define the models formatted as one definition, and one definition only.